requirements.txt

```diff
@@ -1,22 +1,24 @@
 Django==1.8.14 # rq.filter: <1.9
 django-bootstrap3==7.0.1
-requests==2.10.0
+requests==2.11.1
 django-environ==0.4.0
-raven==5.23.0
-elasticsearch==2.3.0
--e git+https://github.com/OpenDataServices/flatten-tool.git@dceac82323271c4cc5956275481d762d81976eaf#egg=flattentool
+raven==5.24.3
+elasticsearch==2.4.0
+-e git+https://github.com/OpenDataServices/flatten-tool.git@dceac82323271c4cc5956275481d762d81976eaf#egg=flattentool-dev
```

Comment:
Yes, it's updated everything to the latest version. Reverting the two .txt files and running `./update_requirements --new-only` should fix this https://github.com/OpenDataServices/grantnav#adding-and-updating-requirements (or we could choose to have requirements updated)